index.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+#container { display: flex; }
+
+nav {
+  padding-top: 10px;
+}
+
+@media only screen and (max-width: 600px) {
+  nav {
+    display: flex;
+    justify-content: space-between;
+    top: 0;
+    width: 100%;
+    position: fixed;
+  }
+
+  .fa-bars {
+    color: blue;
+    padding-right: 20px;
+  }
+
+  .fa-bars:hover {
+    opacity: 0.5;
+  }
+
+  .catch-text-sec {
+    background-image: url(/assets/headerTitle.svg);
+    background-repeat: no-repeat;
+    height: 100vh;
+    background-size: cover;
+  }
+
+  #catch-text-heading {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 50px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    padding-top: 21%;
+    width: 327px;
+    height: 150px;
+  }
```

Comment:
- [ ] Please correct the font attributes `font-size` and `font-height`, and use the `color` to be similar to the Figma template 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/204934493-2d6d9cb4-182d-4c68-a412-f381c4282cc9.png)